### DISCUSSION TITLE
Another way to support this project?

### DISCUSSION THREAD

**Discussion by DavieJMan** [twinbasic/twinbasic#344] [General]
Long time lurker here :-)

First of all, kudos to @WaynePhillipsEA, I am loving this project.

At the moment I'm not ready to commit to a professional/ultimate license as I don't do VB development professionally.  It would be nice to have another, cheaper, option to support this project, with a later discount to upsize to a professional/ultimate license once TwinBasic matures?

e.g.   I would happily donate $10/month to this project without hesitation.  Better if that also came with a future loyalty discount.

I suspect there would be many others here in the same boat?

Just my 2¢...

**Comment by WaynePhillipsEA** [comment]
Thanks!   We certainly appreciate your willingness to help support this project!

Practically, there would have to be enough interest in this in order to justify the administration of another loyalty discount scheme.  If it gets upvotes, we'll consider it.

**Comment by uwbwsvd** [comment]
I am not a professional either. I used VBA for my professional work as an employee. So what I am doing here is hobby. I have never worked with vb. At some point I thought it would be nice to be able to create simple exes with VBA. Now tB came to my attention and I kind of liked it. I bought a licence and consider it as risk capital (there are more expensive hobbies than tB). If tB is more successful than other clones so far, it will be a pleasure for me to have been one of the first.

Translated with www.DeepL.com/Translator (free version)

**Reply by WaynePhillipsEA** [reply]
Thanks for your support @uwbwsvd

**Comment by WaynePhillipsEA** [comment]
Please have a look at the new thread I've opened to discuss the possibility of a new hobbyist tier: #383
